openedx/core/djangoapps/oauth_dispatch/management/commands/edx_clear_expired_tokens.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import unicode_literals
+
+import logging
+from datetime import timedelta
+
+from django.core.exceptions import ImproperlyConfigured
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.utils import timezone
+
```

Comment:
Remove this newline. Third-party imports should be grouped together. See https://www.python.org/dev/peps/pep-0008/#imports.